src/Android/Xamarin.Android/Samples/Data/EditFeatureLinkedAnnotation/EditFeatureLinkedAnnotation.cs

```diff
@@ -0,0 +1,229 @@
+            try
+            {
+                // Create and load geodatabase.
+                string geodatabasePath = DataManager.GetDataFolder("74c0c9fa80f4498c9739cc42531e9948", "loudoun_anno.geodatabase");
+                Geodatabase geodatabase = await Geodatabase.OpenAsync(geodatabasePath);
+
+                // Create feature layers from tables in the geodatabase.
+                FeatureLayer addressFeatureLayer = new FeatureLayer(geodatabase.GeodatabaseFeatureTable("Loudoun_Address_Points_1"));
+                FeatureLayer parcelFeatureLayer = new FeatureLayer(geodatabase.GeodatabaseFeatureTable("ParcelLines_1"));
+
+                // Create annotation layers from tables in the geodatabase.
+                AnnotationLayer addressAnnotationLayer = new AnnotationLayer(geodatabase.GeodatabaseAnnotationTable("Loudoun_Address_PointsAnno_1"));
+                AnnotationLayer parceAnnotationLayer = new AnnotationLayer(geodatabase.GeodatabaseAnnotationTable("ParcelLinesAnno_1"));
+
+                // Create a map with the layers.
+                _myMapView.Map = new Map(BasemapStyle.ArcGISLightGray);
+                _myMapView.Map.OperationalLayers.AddRange(new List<Layer> { addressFeatureLayer, parcelFeatureLayer, addressAnnotationLayer, parceAnnotationLayer });
+
+                // Zoom to the extent of the parcels.
+                await parcelFeatureLayer.LoadAsync();
+                _myMapView.SetViewpoint(new Viewpoint(parcelFeatureLayer.FullExtent));
+            }
+            catch (Exception ex)
+            {
+                new AlertDialog.Builder(this).SetMessage(ex.Message).SetTitle(ex.GetType().Name).Show();
+            }
+        }
+
+        private void MapViewTapped(object sender, GeoViewInputEventArgs e)
+        {
+            // Check if there is already a selected feature.
+            if (_selectedFeature == null)
+            {
+                _ = SelectFeature(e.Position);
+            }
+            else
+            {
+                // Project the user selected point.
+                MapPoint projPoint = GeometryEngine.Project(e.Location, _selectedFeature.Geometry.SpatialReference) as MapPoint;
+
+                // Update the geometry of the selected feature.
+                _ = UpdateGeometry(projPoint);
+            }
+        }
+
+        private async Task SelectFeature(PointF clickedPoint)
+        {
+            try
+            {
+                // Identify across all layers.
+                IReadOnlyList<IdentifyLayerResult> identifyResults = await _myMapView.IdentifyLayersAsync(clickedPoint, 10.0, false);
+                foreach (IdentifyLayerResult result in identifyResults)
+                {
+                    if (result.LayerContent is FeatureLayer layer)
+                    {
+                        _selectedFeature = result.GeoElements.FirstOrDefault() as Feature;
```

Comment:
I recommend either using `First()` if the expectation is that there will always be at least one Feature in the collection, or using `FirstOrDefault` and guarding against a null `_selectedFeature` value.